File: frontend/src/api/platform.js
import request from './request'

// ==================== 租户管理 ====================

// 租户列表
export const getTenantList = (params) =>
  request.get('/platform/tenants', { params })

// 租户详情
export const getTenantDetail = (id) =>
  request.get(`/platform/tenants/${id}`)

// 创建租户
export const createTenant = (data) =>
  request.post('/platform/tenants', data)

// 更新租户
export const updateTenant = (id, data) =>
  request.put(`/platform/tenants/${id}`, data)

// 删除租户
export const deleteTenant = (id) =>
  request.delete(`/platform/tenants/${id}`)

File: backend/routers/v1/platform.go
package v1

import (
	"devops-platform/internal/middleware"
	"devops-platform/internal/modules/user/api"

	"github.com/gin-gonic/gin"
)

func registerPlatformRoutes(v1 *gin.RouterGroup) {
	platform := v1.Group("/platform", authMiddlewares()...)
	{
		tenants := platform.Group("/tenants")
		{
			tenants.GET("", middleware.RequirePermission("tenant", "list"), api.ListTenants)
			tenants.GET("/:id", middleware.RequirePermission("tenant", "list"), api.GetTenantByID)
			tenants.POST("", middleware.RequirePermission("tenant", "create"), middleware.SetAuditOperation("创建租户"), api.CreateTenant)
			tenants.PUT("/:id", middleware.RequirePermission("tenant", "update"), middleware.SetAuditOperation("更新租户"), api.UpdateTenantREST)
			tenants.DELETE("/:id", middleware.RequirePermission("tenant", "delete"), middleware.SetAuditOperation("停用租户"), api.DisableTenantREST)
		}
	}
}
